msdsn/conn_str.go

```diff
@@ -149,6 +150,9 @@ type Config struct {
 	NoTraceID bool
 	// Parameters related to type encoding
 	Encoding EncodeParameters
```

Comment:
>coding EncodeParameters [](http://example.com/codeflow?start=3&length=23)

Please don't use a global variable.
You can add the time zone to `EncodeParameter` which is already being passed around to a lot of places, and if you need to pass it to more places it's not hard to do.

Reply:
I feel the same way. I’ve noticed that global variables can cause uncontrollable side effects. I’ll give it a try tomorrow in UTC+8🤔